cloud/tasks/metrics/empty/counter.go

```diff
@@ -0,0 +1,33 @@
+package empty
```

Comment:
Copied from library/go/core/metrics/nop in order not to depend on library/go/core/metrics